app/views/reservations/your_reservations.html.erb

```diff
@@ -0,0 +1,70 @@
+                                        <h6><%= reservation.start_date.strftime("%v") %></h6>
+                                    </div>
+
+                                    <div class="col-2 mt-3">
+                                            <%= link_to property_path(reservation.property) do %>
+                                                <%= image_tag 'service-img-1.jpg', style: 'width:100%;', class: "img-thumbnail" %>
+                                            <% end %>
+                                    </div>
+
+                                    <div class="col-4 mt-3">
+                                        <h6><%= room.title %></h6>
+                                        <span><%= gravatar_for reservation.user, size: 30 %> <%= link_to reservation.user.name, reservation.user %></span>
+                                    </div>
+
+                                    <div class="col-3 text-right">
+                                        <button type="button" class="btn btn-primary" data-toggle="modal" data-target='<%= "#trigger#{reservation.id}" %>'>
+                                            Review Guest
+                                        </button>
+                                    </div>
+                                    <div class="modal fade" id='<%= "trigger#{reservation.id}" %>' tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+                                        <div class="modal-dialog" role="document">
+                                            <div class="modal-content">
+                                            <div class="modal-header">
+                                                <h5 class="modal-title" id="exampleModalLabel">Review Guest</h5>
+                                                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                                                <span aria-hidden="true">&times;</span>
+                                                </button>
+                                            </div>
+                                            <div class="modal-body">
+                                                <%= form_for @host_params do |f| %>
+                                                    <%= f.hidden_field :reservation_id, value: reservation.id %>
+
+                                                    <div class="form-group">
+                                                        <%= f.text_area :comment, rows: 4, class: "form-control", placeholder: "Comments" %>
+                                                    </div>  
+
+                                                    <div class="modal-footer">
+                                                        <%= f.submit "Add Review", class: "btn btn-primary" %>
+                                                    </div>
+                                                <% end %>
```

Comment:
![Screen Shot 2021-05-27 at 9 10 13 PM](https://user-images.githubusercontent.com/35232348/119831645-0f3e2b00-bf30-11eb-8984-255e24999c16.png)
You're having errors here because you don't have the right variable

As you can see here on the left side of the image
![Screen Shot 2021-05-27 at 9 11 39 PM](https://user-images.githubusercontent.com/35232348/119831739-267d1880-bf30-11eb-970b-3c08ffc3aaa7.png)
You're looking for @host_params but you don't have @host_params in your controller

On the right side, I provided hints on what variables you need.
Its your task now to try and understand this 🙇